docs/source/fitting/fitfunctions/PowerLaw.rst

```diff
@@ -0,0 +1,23 @@
+.. _func-PowerLaw:
+
+================
+PowerLaw
+================
+
+.. index:: PowerLaw
+
+Description
+-----------
+
+A power law background function is defined as:
```

Comment:
lets say something like: "A power law may be used to describe <insert experiment data>"